.github/workflows/build.yml

```
@@ -87,21 +87,9 @@ jobs:
         with:
           filter: |
             Doc/**
-            # Temporarily skip paths with spaces
-            # (i.e. "C API", "Core and Builtins")
-            # to avoid "Error: One of your files includes a space".
-            # Pending https://github.com/python/core-workflow/issues/186
-            # Misc/**
-            Misc/NEWS.d/next/Build/**
-            Misc/NEWS.d/next/Documentation/**
-            Misc/NEWS.d/next/IDLE/**
-            Misc/NEWS.d/next/Library/**
-            Misc/NEWS.d/next/Security/**
-            Misc/NEWS.d/next/Tests/**
-            Misc/NEWS.d/next/Tools-Demos/**
-            Misc/NEWS.d/next/Windows/**
-            Misc/NEWS.d/next/macOS/**
+            Misc/**
             .github/workflows/reusable-docs.yml
+          format: csv  # unbreaks paths /w whitespaces; json is harder to parse
```

Thanks, this one works because we're later checking the list isn't empty:

```yml
steps.changed-docs-files.outputs.added_modified_renamed != ''
```

We're not parsing the output, that's fine.

And this is tested and works in the demo PR: https://github.com/python/cpython/actions/runs/5135145133/jobs/9240164159?pr=105153#step:5:17.

👍 

---

```suggestion
          format: csv  # works for paths with spaces
```